### PR TITLE
Implement new database query

### PR DESCRIPTION
**Pull request by smover**
Add a new query to the database...

### CODE REVIEW

**Comment by cuplvfixr** [comment]
I can help you fixing a bug when using ```Database.query``` in  your `doQuery ` method https://github.com/smover/AwesomeApp/blob/e0dea894bdd7ad93b4a67a116c714ec695d7e921/app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java#L12

Do you want to check it out?

**Comment by smover** [comment]
Yes please

File: app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  public void doQuery(SQLiteDatabase db,
+                      String[] columns) {
+
+    Cursor cursor = db.query("mytable", columns, null, null,
+        null, null, null);
+
+    while (cursor.moveToNext()) {
+      // get stuff from the cursor
+    }
+  }
```
Comment (cuplvfixr) [inline app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java]:
```suggestion
      // @smover: You should eventually call close on the cursor,
      //  I saw more than 3000 people calling close() after calling db.query()
      cursor.close();
    }
```

Comment (smover) [reply app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java]: Show me similar examples

Comment (cuplvfixr) [reply app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java]: Take a look at [this example](https://github.com/hz7225/CEBible/blob/bdec2467acac6dd4e0fd2dac07acd4db233bd47a/src/com/hz7225/cebible/DataBaseHelper.java#L179:L196) :

```java
  /* For SQL query, the low limit for book is 1, not 0 */
  public int getNumOfChapters(int book) throws SQLException {
    // SQL Select Query
    String selectQuery = "SELECT Chapter FROM Bible WHERE Book=" + String.valueOf(book);

    //Log.d(TAG, selectQuery); 
    SQLiteDatabase db = this.getWritableDatabase();
    Cursor cursor = db.rawQuery(selectQuery, null);
    int num;
    if (cursor.moveToLast() == true) {
      num = cursor.getInt(0);
    } else {
      num = 0;
    }
    cursor.close();
    db.close();
    return num;
  }
```

Comment (smover) [reply app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java]: Ok, fix it!

File: app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+
+public class MyDatabase {
+
+  public void doQuery(SQLiteDatabase db,
+                      String[] columns) {
+
+    Cursor cursor = db.query("mytable", columns, null, null,
```
Comment (smover) [inline app/src/main/java/fixr/plv/colorado/edu/awesomeapp/MyDatabase.java:12]:
```suggestion
   // FixrBot change start
   db.doSomethingElse(...);  
   // FixrBot change end
    Cursor cursor = db.query("mytable", columns, null, null,
```

**Pull request closed without merging**